hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.html

```diff
@@ -763,13 +763,12 @@ <h4>Life Insurance</h4>
                       ng-model="entity.health.provider_life_insurance">
                       <ui-select-match
                         prevent-animations
-                        placeholder="Start typing a name or email...">{{$select.selected.label}}</ui-select-match>
-                      <ui-select-choices  repeat="contact.id as contact in contacts.Life_Insurance_Provider | filter: $select.search"
-                        refresh="refreshContacts($select.search, 'Life_Insurance_Provider')"
+                        placeholder="Start typing a name...">{{$select.selected.label}}</ui-select-match>
+                      <ui-select-choices  repeat="provider.value as provider in providers.life_insurance_provider | filter: $select.search"
```

Comment:
Please move the `repeat` to the next line aligned with `refresh` and other props.